Automattic/jetpack
# Subscriptions: update visual appearance of the locked content wall

## Proposed changes:

* Update visual appearance by tweaking block template
* Update translated sentences to not be translated in separate parts; much easier to translate full sentence.
* Add svg lock icon

**Before**

<img width="819" alt="Screenshot 2023-07-13 at 23 18 23" src="https://github.com/Automattic/jetpack/assets/87168/b7a20573-1050-4d0f-95d4-43edf281d398">

**After**

<img width="820" alt="Screenshot 2023-08-02 at 13 05 10" src="https://github.com/Automattic/jetpack/assets/87168/523eaf69-260b-4f94-920a-5dcba9328d23">

Mobile version could be better but that's a matter of optimizing subscribe block, not this paywall:

<img width="406" alt="Screenshot 2023-08-02 at 13 05 16" src="https://github.com/Automattic/jetpack/assets/87168/51a96f0d-e7ba-4aba-80be-22d95655af5b">


### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Create a new post, and mark it "subscribers only"
- As logged out user, see the post at the site
